libraries/radpdfprocessing/overview.md

```diff
@@ -10,49 +10,33 @@ position: 0
 
 # Overview
 
-__RadPdfProcessing__ is a processing library that allows creation, import and export of PDF documents from your code.
-![Rad Pdf Processing Overview 01](images/RadPdfProcessing_Overview_WPF_01.PNG)
+RadPdfProcessing is a processing library that allows you to create, import and export PDF documents from your code. You can use it in any web or desktop .NET application without relying on third-party software like  Adobe Acrobat.
 
->noteIf you still don't have **Telerik Document Processing installed**, check the **[First Steps]({%slug getting-started-first-steps%})** topic to learn how you can obtain the packages through the different suites. 
+In this article, we list the library's most popular features. If you want to learn how to use the library, head straight to the **[Getting Started with RadPdfProcessing]({%slug radpdfprocessing-getting-started%})** article.
 
->noteFor details on the **usage of the library**, go to the **[Getting Started]({%slug radpdfprocessing-getting-started%})** article.
+>noteIf you still don't have **Telerik Document Processing installed**, check the **[First Steps]({%slug getting-started-first-steps%})** topic to learn how to obtain the packages through the different suites with Telerik controls.
 
-The API of RadPdfProcessing contains two different editors, enabling you to choose between editing in a flow-like manner, or using the much more powerful and flexible fixed document structure and drawing on the page.
+The API of RadPdfProcessing contains two different editors, enabling you to choose between editing in a flow-like manner or using the much more powerful and flexible fixed document structure that allows you to draw on the page.
             
 The document model of the library provides support for:
```

Comment:
I think that his bullet list is too long and that its contents fit better in a table.